src/v/config/configuration.h

```diff
@@ -233,8 +233,8 @@ struct configuration final : public config_store {
     // health monitor
     property<std::chrono::milliseconds> health_monitor_tick_interval;
     property<std::chrono::milliseconds> health_monitor_max_metadata_age;
-    property<unsigned> storage_space_alert_free_threshold_percent;
-    property<size_t> storage_space_alert_free_threshold_bytes;
+    bounded_property<unsigned> storage_space_alert_free_threshold_percent;
```

Comment:
I think now you can address one of `TODOs` in this PR mentioning use of bounded property.

Reply:
it is already there, sorry